README.MD

```diff
@@ -42,52 +42,49 @@ touch build.yaml
+| EXTENSIONS   | no       |         | extensions that should be installed for the used tools, currently the following are supported **_[az, google, helm]_**                   | extensions:<br>&nbsp;&nbsp;az:<br>&nbsp;&nbsp;&nbsp;&nbsp;- front-door=1.0.15<br>&nbsp;&nbsp;google:<br>&nbsp;&nbsp;&nbsp;&nbsp;- gsutil<br>&nbsp;&nbsp;&nbsp;&nbsp;- gke-gcloud-auth-plugin<br>&nbsp;&nbsp;&nbsp;&nbsp;- kubectl<br>&nbsp;&nbsp;helm:<br>&nbsp;&nbsp;&nbsp;&nbsp;- https://github.com/databus23/helm-diff=3.5.0<br>&nbsp;&nbsp;&nbsp;&nbsp;- https://github.com/jkroepke/helm-secrets                                                                       |                         |
 
 #### Post Build Config
 
-System configuration and further setups to finish the DMC build afte.
+System configuration and further setups to finish the DMC build.
 
-| ARG                 | Required | Default | Description                                               |
-| ------------------- | -------- | ------- | --------------------------------------------------------- |
-| WORKDIR             | no       | /root   |                                                           |
-| PROFILES            | no       |         | configuration for profile settings                        |
-| POST_BUILD_COMMANDS | no       |         | commands to run after the image build to finish the setup |
+| ARG                 | Required | Default | Description                                               | build.yaml use default | build.yaml setup custom                                                                                                                                |
+| ------------------- | -------- | ------- | --------------------------------------------------------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| WORKDIR             | no       | /root   |                                                           |                        |                                                                                                                                                        |
+| PROFILES            | no       |         | configuration for profile settings                        |                        | profiles:<br>&nbsp;&nbsp;.vimrc:<br>&nbsp;&nbsp;&nbsp;&nbsp;- filetype on<br>&nbsp;&nbsp;.bash_aliases:<br>&nbsp;&nbsp;&nbsp;&nbsp;- alias ll='ls -la' |
```

Comment:
Can you put the "profiles"-code in codeblocks?

Reply:
I can't find a way to this within a table.